be/src/vec/exec/vaggregation_node.cpp

```diff
@@ -459,6 +459,7 @@ Status AggregationNode::open(RuntimeState* state) {
     // this could cause unable to get JVM
     if (_probe_expr_ctxs.empty()) {
         _create_agg_status(_agg_data.without_key);
+        _agg_data_created_without_key = true;
```

Comment:
Could we use _probe_expr_ctxs.empty() as indicator, and do not introduce a new variable _agg_data_created_without_key.